__tests__/app.test.js

```diff
@@ -349,3 +348,45 @@ describe("DELETE /api/comments/:comments_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("Responds with an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const users = body.users;
+        expect(Array.isArray(users)).toBe(true);
+      });
+  });
+  test("Responds with an array of users with correct data types", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body, "<<< GET /api body");
+        const users = body.users;
+        users.forEach((user) => {
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        });
```

Comment:
If you're checking with a forEach then you should also include a length check on the array - if the array is empty then the expects inside the forEach will give you a false positive